Tests/FluentAssertions.Equivalency.Specs/RecordSpecs.cs

```diff
@@ -16,6 +16,16 @@ public void When_the_subject_is_a_record_it_should_compare_it_by_its_members()
         actual.Should().BeEquivalentTo(expected);
     }
 
+    [Fact]
+    public void When_the_subject_is_a_readonly_record_struct_it_should_compare_it_by_its_members()
+    {
+        var actual = new MyReadonlyRecordStruct("foo", new[] { "bar", "zip", "foo" });
```

Comment:
⛏️ (nitpick) Would you mind adding the `// Arrange`, `// Act` and `// Assert` elements?

Reply:
I don't mind, but none of the tests in that file have those marker comments. I really prefer to strive for consistency.